include/deal.II/grid/grid_tools.h

```diff
@@ -756,6 +756,88 @@ namespace GridTools
   std::vector<typename MeshType::active_cell_iterator>
   compute_ghost_cell_halo_layer (const MeshType &mesh);
 
+  /**
+   * Extract and return the active cell within a @p skin_thickness
+   * around a subdomain (set of active cells) in the @p mesh
+   * (i.e. those that share a common set of vertices with the subdomain but
+   * are not a part of it). Here, the "subdomain" consists of exactly all of
+   * those cells for which the @p predicate returns @p true.
+   *
+   * The function first computes interior active, boundary cells of the
+   * subdomain that conform to the predicate.
+   * Using compute_bounding_box, a bounding box is
```

Comment:
I am not sure if doxygen will pick up compute_bounding_box (check this), but I would write

> Using GridTools::compute_bounding_box()...

to be sure.